### ISSUE TITLE
Include comment author's URL in result (formerly: New feature added?)

### ISSUE THREAD

**Issue by weekagooooo323** [philbot9/youtube-comment-scraper#24]
Can you add a new feature to scrape the channel URL of the users' comments.

**Comment by philbot9** [comment]
Yes, that should be possible. The user's URL is being scraped as `authorLink` by the scraping library (see https://github.com/philbot9/youtube-comments-task). It needs to be included in the API response and added to the result on the client-side.
